src/components/PrivacyCheckbox.js

```diff
@@ -1,38 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {FormattedMessage} from 'react-intl';
 
 import {getBEMClassName} from '../utils';
 
 
-const PrivacyCheckbox = ({label, value, onChange}) => {
+const PrivacyCheckbox = ({label, value, warning=false, onChange}) => {
   return (
     <div className={getBEMClassName('privacy-checkbox')}>
-      <input
-        type="checkbox"
-        name="privacy"
-        id="privacy"
-        required={true}
-        value={value}
-        onChange={onChange}
-      />
-      <div className={getBEMClassName('privacy-checkbox__checkmark')}>
-        <label
-          className={getBEMClassName('privacy-checkbox__label')}
-          htmlFor="privacy"
-        >
-          <div
-            className={getBEMClassName('privacy-checkbox__wyswyg-text')}
-            dangerouslySetInnerHTML={{__html: label}}
-          />
-        </label>
+      <div className={classNames(getBEMClassName('privacy-checkbox__input'), {[getBEMClassName('privacy-checkbox__input--warning')]: warning})}>
```

Comment:
This is not very readable, can you break it out of the JSX, and also fix the `getBEMClassName` call? It takes an optional list of modifiers:

```js
const checkboxClassName = getBEMClassName(
  'privacy-checkbox__input',
  warning ? ['warning'] : [],
);
```